code/src/index.css

```diff
@@ -1,13 +1,142 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+    sans-serif; */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #242626;
+  font-size: 14px;
+  font-family: Helvetica, Sans-Serif;
+
 }
 
+
+/* 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+} */
+.outer-wrapper {
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header {
+  width:300px;
+  color: #ffffff;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  gap: 25px;
+}
+
+.albumCard {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+}
+
+.artistName a:not(:first-of-type)::before {
+  color: #a0a0a0;
+  content: ", ";
+}
+
+.coverImage {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+}
+.coverImage img{
+  width: 300px;
+}
+
+.iconsContainer {
+  position: absolute;
+}
+
+.iconsContainer img{
+  width: 40px; 
+   visibility: hidden;
+   padding: 8px;
+   filter: brightness(0) invert(1);
+
+}
+
+
+.albumName {
+  font-weight: bold;
+  padding: 7px 0px;
+}
+
+.albumName a {
+  font-size: 14px;
+  text-decoration: none;
+  color:#ffffff;
+}
+.artistName a{
+  font-size: 14;
+  color: #a0a0a0;
+  text-decoration: none;
+}
+
+
+/* Hover effects  */
+
+.coverImage :not(.iconsContainer):hover{
+  opacity: 0.5;
+}
+
+.coverImage:hover .iconsContainer img{
+  visibility: visible;
+}
+
+.playIcon:hover {
+  width:50px;
+
+}
+
+a:hover {
+  text-decoration: underline;
+  color: #ffffff
+}
+
+
+@media (min-width: 668px) {
+  .header {
+    width: 625px;
+  }
+
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+}
+
+@media (min-width: 1025px) {
+  .header {
+    width: 875px;
+  }
+
+  .container {
+    grid-template-columns: repeat(4, 1fr);  
+  }
+
+  .albumCard {
```

Comment:
The pictures got have been a little bit larger on desktop, since it's a little bit hard to read the text on the album covers.